### DISCUSSION TITLE
Powerful Runners Public Beta Opt-in

### DISCUSSION THREAD

**Discussion by MatthewWEdwards** [community/community#26260] [Actions] [answered]
<p>Hi All,</p>
<p>I noticed that the some new features for GitHub Actions runners have moved into “Public Beta”: <a href="https://github.com/github/roadmap/issues/161" class="inline-onebox" rel="noopener nofollow ugc">Actions: More powerful runners choices for GitHub-managed runners on GitHub.com (Public Beta) · Issue #161 · github/roadmap · GitHub</a></p>
<p>How can I opt-in to the public beta?</p>
<p>Thanks,<br>
Matt</p>

**Comment by diogormendes** [comment]
<p>Did you find something out ?? I would also like to know !!</p>

**Comment by otter-computer** [comment]
<p>Hi <a class="mention" href="/matthewwedwards">@MatthewWEdwards</a> <a class="mention" href="/diogormendes">@diogormendes</a>! I asked the team about this and it’s currently still only in private beta. Sorry about the confusion here, I’ve asked them to update the roadmap to make it clear that this is still in a private testing phase. I’ll make sure to keep you all in the loop when this changes though!</p>

**Comment by JacobReynolds** [comment]
<p><a class="mention" href="/thomasshaped">@ThomasShaped</a> do you know if this has transitioned to a public beta yet?</p>

**Comment by lkfortuna** [comment, marked as answer]
Hi there - this is now a public beta, feel free to sign up here: https://github.com/features/github-hosted-runners/signup

**Reply by tarjei** [reply]
Hi, how do we have to wait toget access if we have signed up?

**Reply by lkfortuna** [reply]
Waitlist is currently sitting at about a 1-week wait time. Thanks for your patience!

**Reply by orilevari** [reply]
Hi there @lkfortuna, do you know if the waitlist is still at about a 1-week wait time? Getting access to runners with more memory is blocking my team's testing strategy. Thanks in advance

**Reply by trash-anger** [reply]
Up! I'm in the same situation right now! This prevent us to get rid of circle-ci for good (although or workflows are already ready and tested for gha)! 🙂

**Reply by orilevari** [reply]
tagging @Akash1134 as I believe @ikfortuna is out on parental leave and Akash seems to be a community manager for github actions. Akash if you see this, could you please advise on the expected wait time to get off the waitlist for public beta?

**Reply by Akash1134** [reply]
@orilevari Thanks for tagging me in. I don't know the exact wait time on top of my head. I've asked the concerned team to share the latest update - I'll let you know once I hear back from them. 🙂

**Reply by orilevari** [reply]
Thank you so much Akash! Looking forward to their response

**Reply by Akash1134** [reply]
@orilevari Are you representing an enterprise or organisation? If so, we can get you enrolled/added right away. Otherwise, we are working on reducing the wait time to less than a day and the feature will be live pretty soon.

**Reply by orilevari** [reply]
Yes, I requested access to the public beta on behalf of my organization, @cloudtostreet. Being added right away would be awesome, thank you 🙂

**Reply by trash-anger** [reply]
I requested access to the public beta on behalf of my organization, @lexop.

**Reply by Akash1134** [reply]
@trash-anger @orilevari Good News! I can confirm that you both were just approved for the public beta by our team internally. Let me know if that's not the case. Thanks!

**Reply by trash-anger** [reply]
Thank you very much! 🙂

**Reply by orilevari** [reply]
Thanks so much! Working for me

**Reply by jakezegil** [reply]
@Akash1134 our team (@tenethq) recently started getting 143/OOM codes which is blocking our deployment - is there any way you would be able to fast track our enrollment?

**Reply by aly-obol** [reply]
@Akash1134 can you please approve ObolNetwork team request to access the large runners?

**Reply by ChiLyMan** [reply]
@Akash1134 We are also running into issues with runner size and this new scalable runner option would be perfect. I've sent through a request for approval into the beta test, but could you please check and see if you can fast track our team (@Bruker-Amics).
